IPython/core/magics/execution.py

```diff
@@ -71,23 +72,27 @@ class TimeitResult(object):
 
     """
 
-    def __init__(self, loops, repeat, best, worst, all_runs, compile_time, precision):
+    def __init__(self, loops, repeat, average, stdev, all_runs, compile_time, precision):
         self.loops = loops
         self.repeat = repeat
-        self.best = best
-        self.worst = worst
+        self.average = average
+        self.stdev = stdev
```

Comment:
Yeah, I wasn't suggesting that the printed representation change. But you can also get the `TimeitResult` object back, and I can imagine wanting to calculate other statistics from the repeated measurements, so it would be useful if it stored all of them.